lib/daemon.js

```diff
@@ -187,9 +188,38 @@ function startDaemon(argv) {
     var nodeArgs = splitCmdLine(nodeArgsStr);
     var stdoutValue = (stdoutBehavior === 'inherit') ? process.stdout : stdoutBehavior;
     var stderrValue = (stderrBehavior === 'inherit') ? process.stderr : stderrBehavior;
+    var listenFd = parseInt(process.env.LISTEN_FD, 10);
+    if (!listenFd) {
```

Comment:
I'm not 100% sure, but I suspect the test here could be wrong.

A non defined LISTEN_FD will result in a NaN value, so still truthy value.
Maybe this should be : 
````js

if (!isNaN(listenFd)) { ..
````